types/genesis.go

```diff
@@ -96,6 +98,11 @@ func (genDoc *GenesisDoc) ValidateAndComplete() error {
 		if len(v.Address) == 0 {
 			genDoc.Validators[i].Address = v.PubKey.Address()
 		}
+
+		if !slices.Contains(acceptedPubKeyTypes, v.PubKey.Type()) {
+			formatStr := "validator %v uses an unsupported pubkey type: %q"
+			return fmt.Errorf(formatStr, v, v.PubKey.Type())
```

Comment:
I'd merge the two lines, as we do in the remaining of the code.

Reply:
I write this sort of code in 2 lines when I think it increases readability.
When there are long error messages, the `fmt.Errof(...)` line is so long that it wraps to the next line.

I could update the other error messages to do this as well (in fact, in my editor all the other messages wrap to the next line, making it a bit awkward to read). What do you think?

Reply:
We don't have a strict policy regarding that, for me honestly is the same. I try to follow what I see more often in the remaining of the code, which is not great but at least preserve some "style".

Really optional here.